### DISCUSSION TITLE
Litestream

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#216] [Showcase]
![litestream io](https://user-images.githubusercontent.com/3902872/113707697-8d284780-96e0-11eb-87aa-284f521ec4ae.jpg)

> Stop building slow, complex, fragile software systems. Safely run your application on a single server. Fully-replicated database with no pain and little cost.

👉 [litestream.io](https://litestream.io/)

**Comment by epogrebnyak** [comment]
Removing the bold font home link in top header menu has a surprisingly good effect on focus on other menu items, did not expect that. Does litestream disclose the source files of website?

**Reply by epogrebnyak** [reply]
But they made removed the boldface item just on first page. Still am intersting move!

The source: https://github.com/benbjohnson/litestream.io
